### PR TITLE
added exception field to schmea/queries

### PR DESCRIPTION
**Pull request by ThorDro**

### CODE REVIEW

File: db/migrate/20210610105453_add_exception_areas.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddExceptionAreas < ActiveRecord::Migration[6.1]
+  def change
+    add_column :areas, :test_exception, :string
```
Comment (bonflintstone) [inline db/migrate/20210610105453_add_exception_areas.rb:3]: Should this not be a boolean? We are communicating with the frontend through json, so booleans are directly supported :)

**Review by bonflintstone — COMMENTED** [review]
And then maybe the naming that I mentioned in the frontend pr

File: db/migrate/20210610105453_add_exception_areas.rb (hunk 1 above)
Comment (ThorDro) [reply db/migrate/20210610105453_add_exception_areas.rb:3]: true, that would be the simplest solution. but it assumes we will never open this up to further options, like dis/en abling the different requirement independly for example, so i went for string as it is easy enough to map 't' to true

Comment (endverbraucher) [reply db/migrate/20210610105453_add_exception_areas.rb:3]: let's keep it simple and use boolean :-)

**Review by bonflintstone — APPROVED** [review]
Nice, much cleaner now :)

**Pull request merged**